packages/morph/src/styles/editor.scss

```diff
@@ -16,7 +16,13 @@ html {
   max-width: 75rem;
   margin: 0 auto;
   padding: 2rem 4rem;
-
+  
+  .cm-cursor {
+    border-left: 2px solid black;
+  }
+  html.dark & .cm-cursor {
```

Comment:
We shouldn't do darkmode like this.

prefer media-queries and set localStorage for `saved-theme`